packages/@okta/vuepress-site/docs/guides/sign-into-web-app/get-user-info/aspnetcore3/getuserinfo.md

```diff
@@ -0,0 +1,25 @@
+ASP.NET Core automatically populates `HttpContext.User` with the information Okta sends back about the user. You can check whether the user is logged in with `User.Identity.IsAuthenticated` in your actions or views, and see all of the user's claims in `User.Claims`.
```

Comment:
switched out logged in for signed in and removed a comma:

ASP.NET Core automatically populates `HttpContext.User` with the information Okta sends back about the user. You can check whether the user is signed in with `User.Identity.IsAuthenticated` in your actions or views and see all of the user's claims in `User.Claims`.